view/frontend/requirejs-config.js

```diff
@@ -7,6 +7,8 @@ var config = {
   map: {
     '*': {
       productProtectionOffer: 'Extend_Integration/js/view/catalog/product/product-protection-offer',
+      simpleProductProtectionOffer:
+        'Extend_Integration/js/view/cart/product-protection-simple-offer',
```

Comment:
Can we change this not to say simple if it's going to be used for all product types?

Reply:
I think "simple" here is in reference to Extend's idea of a "simple offer" not the Magento "simple" product type

Reply:
I was thinking this too but I thought I was maybe ignorant on some Mage jargon.

Reply:
It is `simple` offer from Extend SDK perspective.